### PR TITLE
Added the video and the updated gif in readme file and in docs

### PR DESCRIPTION
**Pull request by SrijanJha28**

### CODE REVIEW

**Comment by Princu7** [comment]
Nice video. Thanks for the request. I would prefer you to write the email address initially instead of doing it in middle. Also, could you make a gif like the existing one. It is super easy. Also record in full screen mode.
http://ricardoalcocer.com/2014/04/adding-videos-readme-file-github-repository/

File: README.md [hunk 1]
```diff
@@ -110,7 +110,10 @@ Or use the 1-click deployment button
 
 ![Generator Screencast](docs/screenshots/screencast.gif)
 
- - Add your **email id**, **name of app** (name of event),
+Video of how to open the Generator Web Form
+https://www.youtube.com/watch?v=Idh4gQI2TO0&feature=youtu.be
+
+- Add your **email id**, **name of app** (name of event),
```
Comment (robbyoconnor) [inline README.md]: email address.

File: README.md [hunk 2]
```diff
@@ -110,7 +110,10 @@ Or use the 1-click deployment button
 
 ![Generator Screencast](docs/screenshots/screencast.gif)
 
- - Add your **email id**, **name of app** (name of event),
+Video of how to open the Generator Web Form
+https://www.youtube.com/watch?v=Idh4gQI2TO0&feature=youtu.be
```
Comment (robbyoconnor) [inline README.md]: Make this a link.

**Comment by Abhi2424shek** [comment]
@SrijanJha28 please follow best practices like **testing first and then making a PR and making a summary of the PR in the PR body** please do that.

**Comment by the-dagger** [comment]
@Princu7 Students are not supposed to write their emails in the Readme.
```Add your email id, name of app (name of event),```
This statement is a part of documentation which guides the users how to use the WebApp generator.

**Comment by Princu7** [comment]
@the-dagger Thanks for the review. When I said write email first, I meant that in the context of gif. That is, in the initial gif, he was first uploading the zip file and then writing his email address in the form. I wanted him to follow the chronological order and enter the email address first and then upload the zip. :smile: Writing email in Readme is certainly wrong.
Also, the existing gif is a little outdated. We just wanted to update it. Would love to hear some of your suggestions. Thanks.

**Comment by the-dagger** [comment]
Got it!
@SrijanJha28 took it in the wrong context then I suppose.
Also for the gif, the gif that he has uploaded is very low both in terms of resolution and quality, so it would be nice if it could be increased by a bit.

**Comment by Princu7** [comment]
@the-dagger No problem at all. Yeah, a slight miscommunication. I totally agree with you. I quality of the gif is really poor and needs to be improved.

**Pull request closed without merging**